docs/basics/userguide/export_training_data.md

```diff
@@ -0,0 +1,165 @@
+[INFO] 16:01: ... trained!
+[INFO] 16:01: Saved ExperimentManager(PPO) using pickle.
+[INFO] 16:01: The ExperimentManager was saved in : 'rlberry_data/temp/manager_data/PPO_2024-06-28_15-58-48_4fc693bc/manager_obj.pickle'
+Backend tkagg is interactive backend. Turning interactive mode on.
+```
+
+
+The easy way to display information is to use {mod}`rlberry.manager.plot_writer_data` as in the [visualization page](visu_userguide), but instead you can use {mod}`rlberry.manager.read_writer_data` function to extract the information into dataframe :
+
+```python
+df = read_writer_data([manager])
+```
+
+Then, you can use your own tools to display whatever you like.
+
+
+To illustrate this, we plot the rollout reward mean of PPO coming from 3 `fit` using matplotlib.
+
+```python
+figure, ax = plt.subplots(1, 1)
+
+for n_simu in df["n_simu"].unique():
+    to_plot_df = df.loc[
+        (df["tag"] == "rollout/ep_rew_mean") & (df["n_simu"] == n_simu),
+        ["global_step", "value"],
+    ]
+    ax.plot(to_plot_df["global_step"], to_plot_df["value"])
+
+ax.set_xlabel("steps")
+ax.set_ylabel("rewards")
+plt.show()
+```
+
+![image](read_writer_example.png)
+
+
+In this previous example, {mod}`rlberry.manager.read_writer_data` had an [ExperimentManager](rlberry.manager.ExperimentManager) as `data_source`, but {mod}`rlberry.manager.read_writer_data` can also take as input a list of ExperimentManager (if you need data on more than one experimentManager), or a path(String) to a directory containing pickle files of an [ExperimentManager](rlberry.manager.ExperimentManager).
+
+
+### Default writer
+
+
+<span>&#9888;</span> Of course, the information contained in the writer (the result of the {mod}`rlberry.manager.read_writer_data` function) depends on how it has been configured and what the agent has recorded in it.<span>&#9888;</span>
+
+In the default writer you have the following information :
+- name : Name of the agent
+- tag : The type/name of the information (depending of the agent login policy, in our previous example it was `rollout/ep_rew_mean` from the [PPO stableBaselines3 agent](https://stable-baselines3.readthedocs.io/en/master/_modules/stable_baselines3/common/off_policy_algorithm.html) )
+- value : The value of the information
+- dw_time_elapsed : Time elapsed since writer initialization
+- global_step : Step where value was added.
+- n_simu : Added by {mod}`rlberry.manager.read_writer_data`, give you the number of the agent (if you use fit>1, you will have information on more than 1 agent in your writer.)
```

Comment:
Maybe : "Added by {mod}`rlberry.manager.read_writer_data`, `n_simu` is an integer identifying the agent (if you use fit>1, you will have information on more than 1 agent in your writer.)"